config.distribution.yaml

```diff
@@ -18,6 +18,7 @@ enable:
   retrieve_cost_data: true
   download_osm_data: true
   download_osm_buildings: false
+  download_osm_buildings_overpass: false
```

Comment:
Interesting proposal, to have a new option here.

What about instead adding an option like: download_osm_method: overpass # or earth_osm? :)
This allows to keep a single retrieve option and it may be clearer. What do you think?
For the rest, this is great :D

Reply:
Thank you for your suggestions, I have tried to implement them as you described. Hopefully, it is fine now :)